intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -216,6 +216,20 @@ \section{First Time connection to the \hpc}
 Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
 \end{prompt}
 
+A possible error message you can get if you previously saved your private key
+somewhere else than the default location (\verb|$HOME/.ssh/id_rsa|):
+
+\begin{prompt}
+%\shellcmd{Permission denied (publickey,gssapi-keyex,gssapi-with-mic).}%
+\end{prompt}
+
+In this case, use the \verb|-i| option for the ssh command to specify the location
```

Comment:
@redfast00 Also `/verb` for `ssh`?

If you also fix that, you can bump the version so this can be merged